layouts/about/list.html

```diff
@@ -31,25 +31,25 @@ <h1>Welcome to {{.Site.Params.company}}</h1>
                   <div class="about__content-wrapper pt-20 pl-85">
                       <div class="section__title mb-30">
                           <div class="section__icon mb-10">
-                              <span class="section__sub-title section__sub-title-2">welcome to sycho</span>
+                              <span class="section__sub-title section__sub-title-2">{{.Params.aboutArea.span}}</span>
```

Comment:
no need to add this about in cms you can make the content dynamic though

Reply:
done